### PR TITLE
Add eslint no var rule

### PR DESCRIPTION
**Pull request by NagaPradeepGajula-osi**
### Jira Story

- [No story INT-NNN](https://osi-cwds.atlassian.net/browse/INT-NNN)

## Description
Adding the no-var eslint rule to encourage the use of const or let instead to be consistent across intake.

## Tests
- [ ] I have included unit tests 
- [ ] I have included feature tests 
- [ ] I have included other tests 
- [x] I have NOT included tests 
Config change only so no tests were included

## Types of changes

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [x] I have ran all tests for the project.
- [x] I have ran lint/rubocop check for the changed/new files.
- [x] My code is in a stable state ready to be deployable, but not necessarily complete.
- [x] I promise on my honor as a CWDS developer to ensure I don't break the build, to check Lint/Rubocop, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -77,6 +77,7 @@
     "no-use-before-define": [2],
     "no-useless-call": [2],
     "no-useless-concat": [2],
+    "no-var": [2],
```
Comment (wmitchell-cambria) [inline .eslintrc]: Can you also add this to the .eslintrc file in our spec/javascript directory?

Comment (NagaPradeepGajula-osi) [reply .eslintrc]: sure. did notice if we had one for spec's

**Comment by NagaPradeepGajula-osi** [comment]
retest this please

**Comment by NagaPradeepGajula-osi** [comment]
retest this please

**Comment by NagaPradeepGajula-osi** [comment]
retest this please

**Comment by NagaPradeepGajula-osi** [comment]
retest this please

**Comment by NagaPradeepGajula-osi** [comment]
retest this please

**Comment by SashaKyz** [comment]
retest this please

**Pull request merged**